synapse/handlers/pagination.py

```diff
@@ -312,8 +314,8 @@ async def purge_room(self, room_id: str) -> None:
     async def get_messages(
         self,
         requester: Requester,
-        room_id: Optional[str] = None,
-        pagin_config: Optional[PaginationConfig] = None,
+        room_id: str,
+        pagin_config: PaginationConfig,
```

Comment:
Looks like these were just wrong?

Reply:
Yup (they date from a time when someone thought that to pass in args as keywords then they needed to be declared with as a keyword)